### ISSUE TITLE
Screen Reader No Longer Reads Characters Such As Left-Brace And Right-Brace

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#38532]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.86.0 (user setup)
- OS Version: Windows 11 Home

Steps to Reproduce:

1. Load a screen reader such as JAWS or NVDA.
2. Open a file (such as a Typescript file) that uses C-style braces `{}`
3. Observe that when reading lines containing braces, the screen reader doesn't announce them. For example, if a line reads `import { lib } from "test";`, the screen reader announces `import lib from "test";`. This is with high punctuation verbosity turned on in the screen reader, so it should be announcing these characters as it does in other applications such as Notepad.
4. Further, when using left and right arrow to read character by character, when arrowing over a brace, the screen reader announces "blank" instead of "left brace" or "right brace."
